go.mod

```diff
@@ -9,8 +9,12 @@ require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/go-logr/logr v0.4.0
 	github.com/golang/mock v1.4.4
+	github.com/onsi/ginkgo v1.15.0
+	github.com/onsi/gomega v1.10.5
 	github.com/openshift/api v0.0.0-20210416115537-a60c0dc032fd
 	github.com/openshift/machine-api-operator v0.2.1-0.20210504014029-a132ec00f7dd
+
+	// k8s 1.18
```

Comment:
This comment isn't up to date

Reply:
Removed it. 

AWS probably needs to be updated as well.